src/components/TopNav.js

```diff
@@ -3,17 +3,21 @@ import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import { Link, Redirect } from 'react-router-dom';
 import {
-  Button,
-  Dropdown,
-  Image,
-  Menu,
-} from 'semantic-ui-react';
+  withStyles,
+} from '@material-ui/core/styles';
+import MenuItem from '@material-ui/core/MenuItem';
+import Menu from '@material-ui/core/Menu';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+import Grid from '@material-ui/core/Grid';
+import { Box } from '@material-ui/core';
 import { FormattedMessage } from 'react-intl';
 import { withCookies, Cookies } from 'react-cookie';
 import { logout as actionLogout } from '@/actions/auth';
 
-import logoImage from '@/assets/images/rovercode_logo_magenta.png';
```

Comment:
I don't have the logo/connection area done from the mocks. That's coming in the next PR.